README.md

```diff
@@ -131,6 +133,36 @@ The `<branded-checkout>` element is where the branded checkout Angular app will
   - `$event.$window` - Provides access to the browser's global `window` object. This allows you to call a custom callback function like `onOrderCompleted` in the example.
   - `$event.purchase` - contains the order's details that are loaded for the thank you page
 
+#### Server-side configuration for a new branded checkout domain
+1. Figure out what domain you will be hosting the branded checkout form on. For example, `myministry.org`
+2. Make sure HTTPS is enabled on that domain
+3. You will need to setup a subdomain for the give.cru.org API. We've experienced CORS issues and cross-domain cookie issues trying to hit the give.cru.org API directly from a custom domain. Create a CNAME record for `brandedcheckout.myministry.org` (the subdomain could be different but using that suggested subdomain makes it consistent with other sites) and point it at `give.cru.org`.
```

Comment:
I think CORS is not an issue; cross-domain cookies are the only issue. FWIW.

Reply:
I can remove that line. It is an issue if you don't configure it at https://maintenance-production-alb-1117343512.us-east-1.elb.amazonaws.com. Lol. Thanks for sharing that again.